app/src/main/res/menu/menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:gravity = "center"
+    tools:context=".activities.MainActivity">
+
+    <item
+        android:id="@+id/favorite"
+        android:orderInCategory="300"
```

Comment:
im not sure what this does, but still hardcoded. there is a resource file for simple int values too